electron_app/helpers/_handleDisk.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+
+const handleFolder = (directory, item) => {
+    let data = {
+            'path': directory,
+            'item': item
+        };
+    return new Promise((resolve, reject) =>{
+        if (!fs.existsSync(directory)) {
+            mkdirp.sync(directory, (err) => {
+                if (err) {
+                    console.log(err);
+                    rejec(err);
```

Comment:
Does this compile? It looks like this should be `reject(err)` not `rejec(err)`.

Reply:
Yes, it's wrong but for some reason is compiling. I'll fix it.